samples/app-in-meeting/csharp/AppInMeeting/AppPackage/manifest.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.9/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.9",
+  "version": "1.0.0",
+  "id": "2901bd2e-6f35-4b83-b512-a1b15e5afa5a",
+  "packageName": "com.teams.appinmeeting",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "App In Meeting",
+    "full": "App In Meeting"
+  },
+  "description": {
+    "short": "App In Meeting",
+    "full": "This example shows app to add before, after and during a meeting"
+  },
+  "accentColor": "#5558AF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://cf40b0aa1718.ngrok.io/configure",
```

Comment:
We can remove the ngrok id here & add placeholder.

Reply:
done